build-system/global-configs/experiments-config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Could you please add comments on what the experiment config template should look like? Thanks.

Reply:
Done!